### PR TITLE
[Keyboard] Add p1800fl

### PR DESCRIPTION
**Pull request by marhalloweenvt**
## Description

This is a project that I work with a local makers for a 1800 Frow-less layout keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/p1800fl/config.h [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x0000
```
Comment (fauxpark) [inline keyboards/handwired/p1800fl/config.h:24]:
```suggestion
#define PRODUCT_ID 0x3EAE
```

```
echo -n "p1800fl" | shasum
```

File: keyboards/handwired/p1800fl/keymaps/default/config.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 marhalloweenvt
```
Comment (fauxpark) [inline keyboards/handwired/p1800fl/keymaps/default/config.h:1]: This file can be deleted.

File: keyboards/handwired/p1800fl/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+void matrix_scan_user(void) {
+
+}
+
+bool led_update_user(led_t led_state) {
+    return true;
+}
+*/
```
Comment (fauxpark) [inline keyboards/handwired/p1800fl/keymaps/default/keymap.c:55]:
```suggestion
```

File: keyboards/handwired/p1800fl/p1800fl.c [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+    // put your per-action keyboard code here
+    // runs for every action, just before processing by the firmware
+
+    return process_record_user(keycode, record);
+}
+*/
+
```
Comment (fauxpark) [inline keyboards/handwired/p1800fl/p1800fl.c:45]:
```suggestion
```

File: keyboards/handwired/p1800fl/config.h (hunk 1 above)
Comment (marhalloweenvt) [reply keyboards/handwired/p1800fl/config.h]: Sorry, but what does this means

```
echo -n "p1800fl" | shasum
3eae5018ebc673f11753103673976a7f4f6ca851  -
```

Do i need to adjust anything for this?

**Comment by marhalloweenvt** [comment]
I have adjusted based on @fauxpark requested. Could you please help me to check them again?

File: keyboards/handwired/p1800fl/config.h (hunk 1 above)
Comment (fauxpark) [reply keyboards/handwired/p1800fl/config.h]: This is just how we generate a pseudo-unique PID when the VID is `0xFEED`. We use the first four digits of the SHA1 hash of the product name without the newline character.

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**